.eslintrc

```diff
@@ -76,10 +76,11 @@
         "camelcase": 2,
         "comma-spacing": 2,
         "comma-style": [2, "last"],
+        "comma-dangle": [2, "always-multiline"],
```

Comment:
I like having dangling commas, because it makes it easier to extend an object later, but I understand if I'm the only one who likes it and this can be removed.